components/SignInfoInputComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import styles from "../styles/SigninPage.module.css";
+
+type props = {
+  name: string;
+  changeEvent: (e: any) => void;
+  validText: string;
+  setValidText: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
이렇게 표현해주는게 일단 좋을 것 같구요!
```suggestion
  setValidText: Dispatch<SetStateAction<string>>;
```

Reply:
setValidText와 같이 상태를 변경할 수 있는 함수를 props로 그대로 내려주는 것 보다 상태를 어떻게 변경할건지를 담고 있는 함수를 내려주는게 더 좋아요! (요건 조금 이해하기 어려울수도 있을 것 같아요..!)